src/containers/simulator/index.js

```diff
@@ -48,7 +48,6 @@ const mapStateToProps = state => ({
   redcode: state.parser.redcode,
   parseResults: state.parser.parseResults,
   standardId: state.parser.standardId,
-  core: state.simulator.core,
```

Comment:
I'd go further, there are a few of these not needed any more, accepting and removing some more